### ISSUE TITLE
Whether there is a plan to support internationalization

### ISSUE THREAD

**Issue by EnochGao** [JetpackDuba/Gitnuro#225]

**Comment by JetpackDuba** [comment]
Yes! I've wanted to implement it for a while to make it more accessible. However, it would require some help from the community to translate in as many languages as possible.

**Comment by EnochGao** [comment]
I can do Chinese translation.🙋

**Comment by JetpackDuba** [comment]
Thank you!

I'll start implementing it after v1.4.0. I'll start by organizing properly the strings and provide some documentation to make it easy for contributors.

**Comment by spipau** [comment]
👍 from my side and a proposal to use [Weblate](https://weblate.org/) to make translations easier without touching code
